R/mod_export_data.R

```diff
@@ -63,6 +63,11 @@ mod_export_data_ui <- function(id) {
           )
         )
       ),
+      fluidRow(column(
+        width = 12,
+        actionButton(ns("selectall"), label = "Select/Deselect all"),
```

Comment:
Name these in detail and use underscores. Instead of just "selectall", name it like "select_all_<attribute>"